domains/auth/Auth.API/Tests/Controllers/TermsControllerTests.cs

```diff
@@ -0,0 +1,139 @@
+using System.Security.Claims;
+using API.Controllers;
+using API.Models;
+using API.Services;
+using API.Utilities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Tests.Controllers
+{
+    public class TermsControllerTests
+    {
+        private readonly TermsController termsController = new TermsController();
+        private readonly IUserService userService = Mock.Of<IUserService>();
+        private readonly IUserDescriptMapper mapper = Mock.Of<IUserDescriptMapper>();
+
+        [Fact]
+        public async Task AcceptTermsAsync_ShouldOnlyUpdateAcceptedTermsVersionAndReturnUser_WhenUserExists()
```

Comment:
This test should just check if the terms version property is updated on the user model sent in as an argument for the method `UpsertUserAsync(...)`.